gitpod-io/gitpod
# [installer]: remove the custom labels from the selector labels

## Description

Remove a [limitation](https://www.gitpod.io/docs/self-hosted/latest/advanced/customization#limitations) on the custom labels by removing the customisation on the selector labels. These are "reverted" to just using the default labels function instead so they're consistent.

## Related Issue(s)

Fixes #11952

## How to test

Deploy via KOTS.
1. Use a default configuration
2. Upload a configuration as per the [docs](https://www.gitpod.io/docs/self-hosted/latest/advanced/customization):
```yaml
customization:
  - apiVersion: "*"
    kind: "*"
    metadata:
      name: "*"
      labels:
        hello: world
      annotations:
        hello: world
```
3. Redeploy with the default configuration

All of these should install fine.

## Release Notes

```release-note
[installer]: remove the custom labels from the selector labels
```

## Documentation


## Werft options:
